lib/yrewrite/forward.php

```diff
@@ -13,14 +13,15 @@ class rex_yrewrite_forward
     public static $pathfile = '';
     public static $paths = [];
 
+    /** @var array|string[] */
     public static $movetypes = [
         '301' => '301 - Moved Permanently',
         '302' => '302 - Found',
         '303' => '303 - See Other',
         '307' => '307 - Temporary Redirect',
```

Comment:
`array|string[]` -> da ist array überflüssig. Am besten geben wir aber gleich einen ganz spezifischen Typen an, mit Typangabe für die Schlüssel.
Dabei fällt mir auf, dass wir hier die Schlüssel als numerische Strings setzen. Die wandelt PHP aber sowieso bei Array in ints um. Daher mein Vorschlag:

```suggestion
    /** @var array<int, string> */
    public static $movetypes = [
        301 => '301 - Moved Permanently',
        302 => '302 - Found',
        303 => '303 - See Other',
        307 => '307 - Temporary Redirect',
```